conda_smithy/configure_feedstock.py

```diff
@@ -88,14 +97,14 @@ def compute_build_matrix(meta):
 
 
 def main(forge_file_directory):
-    recipe_dir = 'recipe'
-    config = {'docker': {'image': 'pelson/obvious-ci:latest_x64', 'command': 'bash'},
+    config = {'is_multi': False,
+              'recipe_dir': "recipe",
+              'docker': {'image': 'pelson/obvious-ci:latest_x64', 'command': 'bash'},
               'templates': {'run_docker_build': 'run_docker_build_matrix.tmpl'},
               'travis': [],
               'circle': [],
               'appveyor': [],
-              'channels': {'sources': ['conda-forge'], 'targets': [['conda-forge', 'main']]},
-              'recipe_dir': recipe_dir}
+              'channels': {'sources': ['conda-forge'], 'targets': [['conda-forge', 'main']]}}
```

Comment:
I'm not sure if that should be kept as the default: It's nice to default there, but unless you wantto grant everybody access to that org I thin this should be set to None both...